str/associatr/pseudobulk.py

```diff
@@ -82,38 +88,23 @@ def pseudobulk(input_file_path, target_sum, min_pct):
 
 
 # inputs:
-@click.option('--input-dir', help='GCS Path to the input AnnData object')
-@click.option('--cell-types', help='Name of the cell type, comma separated if multiple')
-@click.option('--chromosomes', help='Chromosome number eg 1, comma separated if multiple')
-@click.option('--job-storage', help='Storage of the batch job eg 30G', default='8G')
-@click.option('--job-memory', help='Memory of the batch job', default='standard')
-@click.option('--job-cpu', help='Number of CPUs of Hail batch job', default=8)
-@click.option(
-    '--target-sum',
-    help='Target sum of counts per cell (for normalization purposes)',
-    default=1e6,
-)
-@click.option(
-    '--min-pct',
-    help='Minimum percentage of cells expressing a gene to be included in the pseudobulk',
-    default=1,
-)
-@click.command()
-def main(
-    input_dir,
-    cell_types,
-    chromosomes,
-    job_storage,
-    job_memory,
-    job_cpu,
-    target_sum,
-    min_pct,
-):
+
+
+def main():
     """
     Perform pseudobulk (mean aggregation) of an input AnnData object
 
     """
     b = get_batch('Run pseudobulk')
+    input_dir = get_config()['pseudobulk']['input_dir']
+    sample_id_file_path = get_config()['pseudobulk']['sample_id_file_path']
+    cell_types = get_config()['pseudobulk']['cell_types']
+    chromosomes = get_config()['pseudobulk']['chromosomes']
+    job_storage = get_config()['pseudobulk']['job_storage']
+    job_memory = get_config()['pseudobulk']['job_memory']
+    job_cpu = get_config()['pseudobulk']['job_cpu']
+    target_sum = get_config()['pseudobulk']['target_sum']
+    min_pct = get_config()['pseudobulk']['min_pct']
```

Comment:
This is a nit, but you don't have to (and probably shouldn't) read your config up-front. The config file is universally accessible, so instead of 

```python
job_storage = get_config()['pseudobulk']['job_storage']
job_memory = get_config()['pseudobulk']['job_memory']
job_cpu = get_config()['pseudobulk']['job_cpu']
j = b.new_python_job(name=f'Pseudobulk for {cell_type}: chr{chromosome}')
j.cpu(job_cpu)
j.memory(job_memory)
j.storage(job_storage)
```

It's cleaner to use

```python
j = b.new_python_job(name=f'Pseudobulk for {cell_type}: chr{chromosome}')
j.cpu(get_config()['pseudobulk']['job_cpu'])
j.memory(get_config()['pseudobulk']['job_memory'])
j.storage(get_config()['pseudobulk']['job_storage'])
```

This reduces the number of local variables, and makes it clear to a reader that these values are pulled directly from the config. Instead of reading the script and back-tracking to find out if the storage variable was edited after it was read from config, a user can be assured that the value is exactly as it appears in config. 

This is even more evident when an entry is read from config, then passed as an argument (`sample_id_file_path`, `min_pct`) - I'd just pull this from config during the method for overall clarity